CHANGELOG.adoc

```diff
@@ -11,6 +11,7 @@
 - Switch to Sirius Web 2024.5.0
 - Switch to @ObeoNetwork/gantt-task-react 0.4.9
 - Add dependency to org.apache.commons.commons-text 1.10.0
+- Add dependency to Cypress 12.11.0
```

Comment:
Could you also update the github actions please?
The goal is to execute the cypress tests for every PR, as we do in sirius web.

Reply:
done